docs/3-general/instructions-for-use.md

```diff
@@ -1,21 +1,30 @@
-| ISO 13485:2016 Section | Document Section |
-|------------------------|------------------|
-| 4.2.3                  | (All)            |
+<!--
+Copyright (C) 2022 Radiotherapy AI Holdings Pty Ltd
+Copyright (C) 2021-2022 OpenRegulatory (OpenReg GmbH)
+This work is licensed under the Creative Commons Attribution 4.0 International
+License. <http://creativecommons.org/licenses/by/4.0/>.
+
+Original work by OpenRegulatory available at
+<https://github.com/openregulatory/templates>
+-->
 
 # User Manual [enter Product Name]
 
+| ISO 13485:2016 Section | Document Section |
```

Comment:
This changes the location of this ISO reference with respect to the 
# User Manual
entry. 
Is that intentional?  If not intentional, is it acceptable?

Reply:
User Manual itself is a title and not a part of the table. I would be quite surprised if there are any output changes as a result of the markdown auto-formatter...